src/components/asset-equity-text/asset-equity-text.tsx

```diff
@@ -23,6 +23,10 @@ export const AssetEquityText: FC<Props> = ({ asset, style }) => {
   const visibleAmount = formatAssetAmount(new BigNumber(exchangeRate), 2);
   const visibleSymbol = asset.symbol;
 
+  if (!exchangeRate) {
```

Comment:
![image](https://user-images.githubusercontent.com/10281464/142897713-a45cae12-40d3-4aa1-b93f-08e05111a3ae.png)
let's use single style for all conditional rendering (as it's on screenshot)

Reply:
point for code style doc